server/api/batch_tallies.py

```diff
@@ -199,6 +199,9 @@ def validate_batch_tallies_upload(election: Election, jurisdiction: Jurisdiction
     if not jurisdiction.manifest_file_id:
         raise Conflict("Must upload ballot manifest before uploading batch tallies.")
 
+    if jurisdiction.manifest_file.is_processing():
+        raise Conflict("Cannot update batch tallies while manifest file is processing.")
```

Comment:
The user-facing name for batch tallies is "candidate totals by batch" so we should update all error messages to use that language. And I would also replace "manifest file" with "ballot manifest"

Reply:
[8ac01cb](https://github.com/votingworks/arlo/pull/2043/commits/8ac01cb173a94a26a6a4d3b10e73304bb9eabcf4)